config/jobs/kubernetes/sig-k8s-infra/releng/artifact-promotion-presubmits.yaml

```diff
@@ -8,7 +8,7 @@ presubmits:
       testgrid-dashboards: sig-k8s-infra-k8sio
     decorate: true
     skip_report: false
-    run_if_changed: 'k8s.gcr.io/((images/.*/images\.yaml)|(manifests/.*/promoter-manifest\.yaml))'
```

Comment:
Can you revert all the changes to this file please? This will break the image promotion jobs right now. I will resolve this as part of the image registry freeze in March.